frontend/src/pages/org/workflow-editor.ts

```diff
@@ -1908,6 +1965,9 @@ https://archiveweb.page/images/${"logo.svg"}`}
   private updateFormStateOnChange(e: Event) {
     const elem = e.target as SlTextarea | SlInput | SlCheckbox;
     const name = elem.name;
+    if (!this.formState.hasOwnProperty(name)) {
+      return;
+    }
```

Comment:
Is this something you've observed/expect, or just defensive in case?

Reply:
Good question, this is sort of left over from stricter typechecking--probably will refactor it out in https://github.com/webrecorder/browsertrix-cloud/issues/1204